src/main/java/frc/robot/subsystems/lifter/ArmSubsystem.java

```diff
@@ -69,7 +74,7 @@ public Command moveToCommand(TrapezoidProfile.State goal) {
         return new FunctionalCommand(
             () -> initalizeMoveTo(goal),
             () -> executeMoveTo(),
-            (interrupted) -> {},
+            (interrupted) -> CommandScheduler.getInstance().schedule(keepInPlaceCommand()),
```

Comment:
No this is not the right way to do this and will almost certainly cause problems. When you start calling schedule within commands, weird stuff happens and its hard to debug. You should use a trigger, and call on true when the trigger happens. If you want to have the trigger happen only when you've run a particular command, have that command set an instance variable in the subsystem indicating its running.

Reply:
```
Trigger isAtGoal() {
  return isAtPosition(m_goal.position) &&
isAtVelocity(m_goal.velocity) && //optional
m_goal_active // set to true at the beginning of close looped commands, false at the beginning of open loop commands
}
```

In `RobotContainer`:

```
m_elevator.isAtGoal(keepInPlaceCommand());
```